### PR TITLE
Update to wp-env 8

### PR DESCRIPTION
**Pull request by lithrel**
- Update to @wordpress/env 8.0.0
  - Fixes file permissions issues
  - Has grouped containers for cli/wp/composer
  - Disallow unknown config keys
- Change in content mapping and paths
- Change in functions to run wp, composer, cli commands
- wp-env version check
  - unexpected version is marked with a red cross
- change config .p4-env.json to .wp-env.json merge strategy
  - external config keys moved to .p4-env.json, override for both config used
- fix on CI E2E results export
- composer run-script site:custom works

## Update instructions

- Save your work (the update process will destroy everything)
- Destroy everything `npm run env:destroy`
- Update this repo
- Update wp-env package `npm -g install @wordpress/env@8.0.0`
- Reinstall :man_dancing:

### CODE REVIEW

**Comment by comzeradd** [comment]
Should we also adjust [wpenvCheck](https://github.com/greenpeace/planet4-develop/blob/feature/wpenv-update-7/scripts/lib/env-check.js#L14) since we are upgrading to wp-env 8?

Also when running `npm run env:requirements`, I get this message:

```bash
✓ 5.16.0
This installation process requires wp-env version 7. Version <8> is not officially supported.
```

Maybe we can add a red X icon in front of the second raw, or something else as a visual hint that something is wrong.

**Comment by lithrel** [comment]
> Should we also adjust [wpenvCheck](https://github.com/greenpeace/planet4-develop/blob/feature/wpenv-update-7/scripts/lib/env-check.js#L14) since we are upgrading to wp-env 8?

oops :facepalm: fixed

File: .p4-env.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "planet4": {
+    "content": {
+      "db": "v0.2.18",
+      "images": "1-25"
+    },
+    "repos": {
+      "planet4-base": "main",
+      "planet4-master-theme": "main",
+      "planet4-plugin-gutenberg-blocks": "fix/columns-tests-timing"
```
Comment (comzeradd) [inline .p4-env.json:10]: Should we reset the branch here?

Comment (lithrel) [reply .p4-env.json:10]: right, just waiting for [that pr](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1062) to be merged to give this test a better chance to pass :grimacing:

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "planet4-develop",
-  "version": "v0.4",
+  "version": "v0.5",
```
Comment (comzeradd) [inline package.json:3]: That may require a similar change to the lock file.

**Review by comzeradd — APPROVED** [review]
Tested locally too and everything works great :+1:

**Pull request merged**